### PR TITLE
feat: add storybook and figma pages

### PR DESCRIPTION
**Pull request by peter-zawistowicz**
# PR Description

### Summary of my changes and explanation of my decisions

- adds page for Storybook addon 
- adds page for Figma plugin

-----
<a href="https://stackblitz.com/~/github/stackblitz/docs/tree/peter-zawistowicz%2Fstorybook">Review PR in StackBlitz</a> _Submitted with [StackBlitz](https://stackblitz.com/~/github/stackblitz/docs/tree/peter-zawistowicz%2Fstorybook)._

### CODE REVIEW

**Comment by peter-zawistowicz** [comment]
@sulco updated the images and Figma install instructions. ready for another 👀

**Review by HeyGarrison — APPROVED** [review]
This LGTM!

File: docs/guides/integration/storybook-addon.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+Storybook supports a number of [addons](https://storybook.js.org/docs/addons) that extend the functionality and integrate additional tools.
+
+::: tip Available with all StackBlitz plans including **Personal**, **Personal+**, **Teams**, and **Enterprise**.
+
+The addon is compatible with all Storybook deployments including for Chromatic customers.
+:::
+
+## About the StackBlitz addon for Storybook
+
+With StackBlitz’s addon, Storybook users can open stories in StackBlitz directly from Storybook.
```
Comment (sulco) [inline docs/guides/integration/storybook-addon.md:20]: I'd consider linking the addon already here as well, for the more hot-headed readers ("ok, I know what it does, just give me the addon")

File: docs/guides/integration/figma-plugin.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Commit edits back to the repository or open a pull request
+
+## **Installing the plugin**
+
+To use the StackBlitz plugin, add it to a Figma file from either:
+
+- **The [plugins directory](https://www.figma.com/community/plugin/1287113487428429346)** by clicking “Open in…” and then selecting the name of the file.
+    
+    ![Figma plugin directory](./assets/figma-installation.gif)
+    *Initialize the StackBlitz plugin from within Dev Mode.*
```
Comment (sulco) [inline docs/guides/integration/figma-plugin.md:39]: I think this should go under the second bullet point, right?

File: docs/guides/integration/figma-plugin.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+
+- **Dev Mode within your Figma file** by searching for "StackBlitz" under *Plugins*
+
+## Using the StackBlitz plugin with Dev Mode
+
+Once you have added the plugin to your Figma file,
+
+- Enable [Dev Mode](https://www.figma.com/dev-mode/) is on (`Shift` + `D`)
```
Comment (sulco) [inline docs/guides/integration/figma-plugin.md:48]: minor: changing it to a numbered list will emphasize that these are the steps that each follow after the other

**Review by sulco — COMMENTED** [review]
looks good, reads great! Just left some comments about minor adjustments. After these we're good to go!

**Review by peter-zawistowicz — COMMENTED** [review]
final fixes!

**Pull request merged**